src/pybind/mgr/rgw/module.py

```diff
@@ -374,5 +378,7 @@ def import_realm_token(self,
                            placement: Optional[str] = None,
                            start_radosgw: Optional[bool] = True,
                            zone_endpoints: Optional[str] = None) -> None:
-        self.rgw_zone_create(zone_name, realm_token, port, placement, start_radosgw,
+        port = int(port) if port else None
+        placement_spec = placement.get('placement') if placement else None
+        self.rgw_zone_create(zone_name, realm_token, port, placement_spec, start_radosgw,
```

Comment:
Same, if you follow the previous recommendations there's no need to these changes.